packages/design-system/Button.tsx

```diff
@@ -1,9 +1,11 @@
-'use client'
+'use client';
 
-import * as React from 'react'
+import * as React from 'react';
 
 export function Button() {
   return (
-    <button className="bg-blue-500 hover:bg-blue-700 h-16 w-16">Boop</button>
-  )
+    <button type="button" onClick={() => alert('boop')}>
```

Comment:
remove this inline callback function, instead accept this as a prop

Reply:
done